### DISCUSSION TITLE
Create Android Bundle (.aab) so raylib app could be uploaded to PlayStore

### DISCUSSION THREAD

**Discussion by Maor2871** [raysan5/raylib#2766] [Features]
The [raylib to apk guide](https://github.com/raysan5/raylib/wiki/Working-for-Android) is great, but many developers would like to take their application to the next level and publish it in the android play store.
For that, we need to export our raylib application to an app bundle (.aab) file, because google play console does not allow anymore to upload apps as .apk file.
I believe the process of making the .aab file is mostly technical and not requires any further development, all the raylib users need for that is a new guide, just like the apk guide.

**Comment by raysan5** [comment]
@Maor2871 Feel free to update it, the Wiki is open for editing.
